apps/site/assets/ts/stop/__tests__/StopPageOverlayTest.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import StopPageDepartures from "../components/StopPageDepartures";
+import { Route, RouteType, Stop } from "../../__v3api";
+import StopPageOverlay from "../components/StopPageOverlay";
+import { shallow } from "enzyme";
+import { ScheduleWithTimestamp } from "../../models/schedules";
+
+const baseRoute = (name: string, type: RouteType): Route =>
+  ({
+    id: name,
+    direction_destinations: { 0: "Somewhere there", 1: "Over yonder" },
+    name: `${name} Route`,
+    type
+  } as Route);
+const stop = {} as Stop;
+const routeData: Route[] = [baseRoute("4B", 3), baseRoute("Magenta", 1)];
+const scheduleData = [] as ScheduleWithTimestamp[];
+describe("StopPageOverlay", () => {
+  it("opens on click", () => {
+    // TODO update test after departure card interaction is setup
+    const wrapper = shallow(
```

Comment:
suggestion (non-blocking): Could you re-write this using [react-testing-library ](https://testing-library.com/docs/ecosystem-user-event/#api)instead of enzyme? I see a [ticket](https://app.asana.com/0/385363666817452/1203355780667458/f) to remove enzyme tests, so it may be worth avoiding adding new ones!